api/src/pcapi/routes/public/individual_offers/v1/bookings.py

```diff
@@ -15,6 +18,58 @@
 from . import bookings_serialization as serialization
 
 
+def _get_bookings_query(offer_id: int) -> sqla_orm.Query:
+    return (
+        booking_models.Booking.query.join(offerers_models.Venue)
```

Comment:
C'est un requête qu'on va appeler très souvent et on va essayer de la rendre efficace. 
Avec les join que tu as utilisé tu charges de la db toutes les colonnes de toutes les tables que tu join. Je doute qu'on ait besoin de offer.description qui peut être lourde par exemple.
Dans ce cas ci on peut avoir en plus 50 lignes retournées par la requête. 

Tu peux le faire dans un ticket ensuite mais il faudrait essayer d'utiliser des trucs du genre : https://github.com/pass-culture/pass-culture-main/blob/6c1419bd2e703945317760ff2f0475e6574f4bdb/api/src/pcapi/core/offerers/repository.py#L254

Reply:
On devrait sans doute le faire ensuite pour toutes les autres routes

Reply:
voici la requête exécutée:
`SELECT booking.id AS booking_id,
    booking."dateCreated" AS "booking_dateCreated",
    booking."dateUsed" AS "booking_dateUsed",
    booking."stockId" AS "booking_stockId",
    booking."venueId" AS "booking_venueId",
    booking."offererId" AS "booking_offererId",
    booking.quantity AS booking_quantity,
    booking.token AS booking_token,
    booking."userId" AS "booking_userId",
    booking.amount AS booking_amount,
    booking."priceCategoryLabel" AS "booking_priceCategoryLabel",
    booking."cancellationDate" AS "booking_cancellationDate",
    booking."displayAsEnded" AS "booking_displayAsEnded",
    booking."cancellationLimitDate" AS "booking_cancellationLimitDate",
    booking."cancellationReason" AS "booking_cancellationReason",
    booking.status AS booking_status,
    booking."reimbursementDate" AS "booking_reimbursementDate",
    booking."depositId" AS "booking_depositId"
FROM booking
    JOIN venue ON venue.id = booking."venueId"
    JOIN venue_provider ON venue.id = venue_provider."venueId"
    JOIN stock ON stock.id = booking."stockId"
    JOIN offer ON offer.id = stock."offerId"
WHERE venue_provider."providerId" = %(providerId_1) s
    AND venue_provider."isActive" = true
    AND offer.id = %(id_1) s
ORDER BY booking."dateCreated" DESC`
On ne récupère rien de offer ni de stock ou venue à part les id

Reply:
Alors dans ce cas ce qui m'étonne c'est que je voit pas de offer.name ou offer.id (qui sont utilisés dans la réponse)

Reply:
J'ai fait un print depuis les tests, voici le json que j'ai en sortie:
`{'bookings': [{'confirmationDate': '2023-06-29T10:04:42.878140Z',
               'creationDate': '2023-06-29T10:04:42.878140Z',
               'id': 5988,
               'offerEan': '1234567890123',
               'offerId': 23609,
               'offerName': 'Vieux motard que jamais',
               'price': 1010,
               'priceCategoryId': None,
               'priceCategoryLabel': None,
               'quantity': 1,
               'stockId': 13789,
               'userBirthDate': '2005-05-03',
               'userEmail': 'beneficiary@example.com',
               'venueAddress': '1 boulevard Poissonnière',
               'venueDepartementCode': '75',
               'venueId': 27215,
               'venueName': 'Le Petit Rintintin 0'}],
 'pagination': {'currentPage': 1,
                'itemsCount': 1,
                'itemsTotal': 1,
                'lastPage': 1,
                'limitPerPage': 50,
                'pagesLinks': {'current': 'http://localhost/public/bookings/v1/bookings?page=1&limit=50',
                               'first': 'http://localhost/public/bookings/v1/bookings?page=1&limit=50',
                               'last': 'http://localhost/public/bookings/v1/bookings?page=1&limit=50',
                               'next': None,
                               'previous': None}}}
`